app/src/main/java/com/example/andresarango/aughunt/login/SplashScreen.java

```diff
@@ -11,5 +13,16 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_splash_screen);
+
+
+        Handler handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                System.out.println("SPLASHING WAITING... 5s");
+                startActivity(new Intent(SplashScreen.this, StartActivity.class));
+                finish();
+            }
+        }, 5000);
```

Comment:
what happens if the user hits the back button before the 5 seconds are over? ;-)
hint: you probably want to move this to onStart and call handler.removeCallbacks() in onStop.